weblate/formats/ttkit.py

```diff
@@ -176,6 +177,15 @@ def flags(self):
             flags.merge(self.template.xmlelement)
         return flags.format()
 
+    def clone_template(self) -> None:
+        new_unit = copy(self.template)
+
+        # do not copy notes from the template (#11133)
+        new_unit.removenotes()
+
+        self.mainunit = self.unit = new_unit
+        self._invalidate_target()
```

Comment:
Please avoid duplicating logic from the super class:

```suggestion
        super().clone_template()

        # do not copy notes from the template (#11133)
        self.unit.removenotes()
```

Reply:
Fair :)